datumaro/datumaro/util/tf_util.py

```diff
@@ -35,16 +35,23 @@ def check_import():
 def import_tf(check=True):
     import sys
 
-    tf = sys.modules.get('tensorflow', None)
-    if tf is not None:
+    not_found = object()
```

Comment:
Is it performance optimisation? Could you please add a comment?

Reply:
No, it is a way to be sure about the result of `get()` call. The problem with default `None` is that it is a valid value for a module, and, moreover, we need it later. Using some specific `object()` we can be sure about existence of key and its value.